voice:
    gender: 'female'
    gender_translation_hints:
        'female': 'She said'
        'male': 'He said'
        'unknown': 'They said'
    sound_packages: [ ]
    lang: 'ru,en'
    default_profile: 'coqui-ai'
    ladspa_path: ~/.ladspa/
    voice_profile:
        coqui-ai:
            type: 'coqui-ai'
            langs: [ 'en', 'es', 'fr', 'de', 'it', 'pt', 'pl', 'tr', 'ru', 'nl', 'cs', 'ar', 'zh', 'hu', 'ko', 'ja', 'hi' ]
            enabled: true
            coqui_ai_params:
                speaker: "Asya Anara"
